src/main/java/sparksoniq/config/SparksoniqRuntimeConfiguration.java

```diff
@@ -50,6 +46,47 @@ public String getConfigurationArgument(String key) {
             return null;
     }
 
+    public String getOutputPath() {
+        if (this._arguments.containsKey("output-path"))
+            return this._arguments.get("output-path");
+        else
+            return null;
+    }
+
+    public String getLogPath() {
+        if (this._arguments.containsKey("log-path"))
+            return this._arguments.get("log-path");
+        else
+            return null;
+    }
+
+    public String getQueryPath() {
+        if (this._arguments.containsKey("query-path"))
+            return this._arguments.get("query-path");
+        else
+            return null;
+    }
+
+    public int getResultSizeCap() {
+        if (this._arguments.containsKey("result-size"))
+            return Integer.parseInt(this._arguments.get("result-size"));
+        else
+            return 200;
+    }
+
+    public boolean isShell() {
+        System.out.println(_arguments.get("shell"));
```

Comment:
I think this prints a yes, right before the logo in shell.
```
yes
   _____                  __                    ________
  / ___/____  ____ ______/ /___________  ____  /  _/ __ \
  \__ \/ __ \/ __ `/ ___/ //_/ ___/ __ \/ __ \ / // / / /
 ___/ / /_/ / /_/ / /  / ,< (__  ) /_/ / / / // // /_/ /
/____/ .___/\__,_/_/  /_/|_/____/\____/_/ /_/___/\___\_\
Item Display Limit: -
Output Path: -
Log Path: -
Query Path : -
```

Reply:
Also prints null for the not given case.
```
null
************************
Sparksoniq version 0.9.7
************************
Usage:
spark-submit <Spark arguments> <path to sparksoniq jar> <Sparksoniq arguments>

Example usage:
spark-submit sparksoniq-0.9.7.jar --shell yes
spark-submit --master local[*] sparksoniq-0.9.7.jar --shell yes
spark-submit --master local[2] sparksoniq-0.9.7.jar --shell yes
spark-submit --master local[*] --driver-memory 10G sparksoniq-0.9.7.jar --shell yes

spark-submit --master yarn sparksoniq-0.9.7.jar --shell yes
spark-submit --master yarn --executor-cores 3 --executor-memory 5G sparksoniq-0.9.7.jar --shell yes
spark-submit --master local[*] sparksoniq-0.9.7.jar --query-path my-query.jq
spark-submit --master local[*] sparksoniq-0.9.7.jar --query-path my-query.jq
spark-submit --master yarn --executor-cores 3 --executor-memory 5G sparksoniq-0.9.7.jar --query-path hdfs:///my-query.jq --output-path hdfs:///my-output.json
spark-submit --master local[*] sparksoniq-0.9.7.jar --query-path my-query.jq --output-path my-output.json --log-path my-log.txt
2019-05-29 18:36:37 INFO  ShutdownHookManager:54 - Shutdown hook called
2019-05-29 18:36:37 INFO  ShutdownHookManager:54 - Deleting directory /private/var/folders/lp/_z0splp10696bz6hlw2t31_w0000gn/T/spark-f5317629-5f6f-4fea-bb6d-395309b3d6e6
```

Reply:
Good catch, thanks!